### ISSUE TITLE
Can't revert back to the original steam play filter behavior after enabling steam play for other titles. 

### ISSUE THREAD

**Issue by BigCojones** [ValveSoftware/steam-for-linux#10769]
#### Your system information

* Steam client version (build number or date):  6th of April 2024
* Distribution (e.g. Ubuntu): Zorin lite 16.3
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Up to date
* Steam Logs: [generate by running this command in a terminal `tar -zcvf ~/Desktop/steam-logs.tar.gz ~/.steam/steam/logs`]
* GPU: Nvidia/Intel/AMD Intel

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/)

So i'm running some older hardware and was testing the gaming capabilities, protonup-qt, luxtorpeda, wined3d etc. The results weren't that great that i would want to miss the steam play filter which sorts the native linux games as that's the easiest way to search for games that run well on this hardware. 

After playing around a bit i notice the steam play filter isn't working with the expected behavior. I would have assumed that just by disabling "steam play for all other titles" would revert it back, but it did not. 

I had to manually set every game that used any compat to not use it, to even able to uninstall proton and the steam run time, but that didn't revert it either. I have had protonup uninstalled and the folders it created deleted. I have deleted steam itself and that didn't help, all of this with the "steam play for other titles" disabled the whole time. When i reinstalled steam it automatically just started downloading proton-exp and the steam run time again and the filter doesn't work.

Is there really no way of reverting back and getting back the original functionality of the steam play filter?! Next i'll try uninstalling steam and all the persistent files it leaves and if that doesn't work, a full system reinstall seems like the only way.

You would expect it to revert back to the original behavior just by disabling the "Steamplay for all other titles" from the settings. But it clearly does not. It feels like i'm stuck with option on now, as it won't even let me be without proton and steam run time 3.0 now...

#### Steps for reproducing this issue:

If you can get back the original steam play filter behavior where it only shows the real native titles. You have cracked the code. :D
But from what I understand it isn't even possible.

**Comment by Alteration** [comment]
**Your system information**

- Steam client version (build number or date): Steam Version 17165846667
- Distribution (e.g. Ubuntu): EndeavorOS (Arch-Linux)
- Opted into Steam client beta?: Yes (After encountering this issue)
- Have you checked for system updates?: Yes
- GPU: nVidia RTX 3070 Ti

I was experiencing this same thing. Toggled on "Enable Steam Play for all other titles" assuming it meant I would have the _option_ to try make them compatible while the filter would still list only those that were officially compatible.

I can't say its a real solution but I was able to get back to the original functionality by running rm -rf on ~/.steam/steam/ to wipe the linked directory and then rm -rf on ~/.steam/

I ran those while steam was closed, then the next time I opened steam it started downloading again. I can't say for certain how this will affect others, I'm on basically a fresh install.

This

**Comment by BigCojones** [comment]
Yeah i had to do something similar. Removed and reinstalled steam to get it back. Maybe even purge. It was a while ago, so can't remember.

But yeah, it sucks when most tutorials to get proton running first say to enable steam play for all titles. This move is completely unnecessary and makes this problem. If you don't enable it for all, you keep the original filter and then can manually enable by a game-by-game basis.
